frontend/src/app/useApplicationSettings.tsx

```diff
@@ -32,11 +31,10 @@ export const useApplicationSettings = (): {
             // NOTE: this endpoint only requests ouath because of the security layer, this is not an ironclad use-case
             // Something went wrong on the server with the Oauth, let us just log them out and refresh for them
             console.error(
-              'Something went wrong with the oauth token, logging out...',
+              'Something went wrong with the oauth token, please log out...',
               e.message,
               e,
             );
-            logout().then(() => window.location.reload()); // note this is a bad side-effect, never do this
```

Comment:
This logic was specifically requested to refresh the client. @cfchase requested it for 1.16/1.17 involvement. We should probably look at a way to do this in some fashion at least once.

Reply:
ok, great, let’s see what he says about this!

Reply:
This was a backstop because we were stuck at a screen and had no way of logging out.  Is it causing an issue?